packages/client/src/ui/Inventory.tsx

```diff
@@ -26,7 +26,7 @@ export function Inventory() {
           hero: { url: string; creature: number; image: string },
           index: number
         ) => (
-          <div key={index}>
+          <div key={index} onClick={() => setAcHeroFn(hero)}>
```

Comment:
can you switch to `onMouseEnter` rather than `onClick`

Reply:
onclick can show and cancel

Reply:
It's ok